Tasks/AzureFileCopy/task.json

```diff
@@ -1,4 +1,5 @@
 ﻿{
+<<<<<<< HEAD
```

Comment:
did these come because of a conflict in cherry-pick?
Please update PR after fixing these.